modules/markup/renderer.go

```diff
@@ -163,6 +166,18 @@ type nopCloser struct {
 
 func (nopCloser) Close() error { return nil }
 
+func renderIFrame(ctx *RenderContext, renderer Renderer, input io.Reader, output io.Writer) error {
+	_, err := io.WriteString(output, fmt.Sprintf(`<iframe src="%s%s/%s/render/%s/%s" name="ifd"
+onload="this.height=ifd.document.body.scrollHeight" width="100%%" scrolling="no" frameborder="0"/>`,
```

Comment:
`scrolling=no` is apparently deprecated and should be replaced (or accompanied) by `style="overflow: hidden"`, which however does not appear to be supported on all browsers.
Should we supply both (or add an additional style class)?

Reply:
I will add `style="overflow: hidden"`, split style to a file looks like not a good idea because it's a embed iframe.